public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -594,871 +653,773 @@ function createSidebar() {
+      const toggles = $('.BLtoggle');
+
+      if (toggles[0]) {
+        const btns = toggles.find('button, a');
+
+        btns.click(function (e) {
+          if (!e.target.href)
+            e.preventDefault();
+          const theme = $(this).attr("data-color");
+          if (!theme)
+            return;
+          btns.removeClass('active');
+          btns.filter('a[data-color="' + theme + '"]').addClass('active');
+          btns.filter('button[data-color="' + theme + '"]').addClass('active');
+
+          doBeeline(theme, theme);
+        });
+      }
+    }
+    /*$("#doBeeLine").on("click", function () {
+    		
+	    	const theme = activateBeeLine.theme !== 'bright' ? 'bright' : 'off';
+    		
+    		const toggles = $('.BLtoggle');
+    		
+    		if (toggles[0]) {
+    			const btns = toggles.find('button, a');
+	    		btns.removeClass('active');
+    			btns.filter('a[data-color="' + theme + '"]').addClass('active');
+    			btns.filter('button[data-color="' + theme + '"]').addClass('active');
+	    	}
+		    activateBeeLine.theme = theme;
+    		event.preventDefault();
+    		doBeeline(theme, theme);
 	});
+*/
+  }
+
+
+
+
+  $("#size").change(function () {
+
+    var initial_data = $(this).val();
+
+
+    //CHANGE CSS TO SIZE FUNC VALUE
+    $('section.mt-content-container p').css("font-size", initial_data + "rem");
+    $("#size").val(initial_data);
+
+    //LOG SIZE VALUE AFTER INPUT DATA
+
+    localStorage.setItem('font_size', initial_data);
+
+
+
+  });
+
+  $("#slider-page-width").change(function () {
+    var initial_data = $(this).val();
+
+    $('section.mt-content-container').css("margin-left", initial_data + "px");
+    $('section.mt-content-container').css("margin-right", initial_data + "px");
+
+    $("#slider-page-width").val(initial_data);
+
+
+    localStorage.setItem('page_width', initial_data);
+    console.log("change" + localStorage.getItem('page_width'));
+  });
+
+
+  $('body').click(function (event) {
+    if (!$(event.target).closest('#custom_sidebar').length && !$(event.target).is('#custom_sidebar')) {
+      document.getElementById("custom_sidebar").style.width = "0";
+
+    }
+  });
+  $('body').click(function (event) {
+    if (!$(event.target).closest('#sidebar1').length && !$(event.target).is('#sidebar1')) {
+      document.getElementById("sidebar1").style.width = "0";
+
+    }
+  });
+  $('body').click(function (event) {
+    if (!$(event.target).closest('#sidebar2').length && !$(event.target).is('#sidebar2')) {
+      document.getElementById("sidebar2").style.width = "0";
+
+    }
+  });
+  $('body').click(function (event) {
+    if (!$(event.target).closest('#sidebar3').length && !$(event.target).is('#sidebar3')) {
+      document.getElementById("sidebar3").style.width = "0";
+
+    }
+  });
+
+  $('body').click(function (event) {
+    if (!$(event.target).closest('#sidebar4').length && !$(event.target).is('#sidebar4')) {
+      document.getElementById("sidebar4").style.width = "0";
+
+    }
```

Comment:
One more click listener section that is either redundant or at least can be consolidated.